addons/search-profile/userscript.js

```diff
@@ -7,19 +7,36 @@ export default async function ({ addon, console, msg }) {
     user = document.querySelector('[name="q"]').value.trim(),
     valid = /^[\w-]{3,20}$/g.test(user);
   tab.type = "button";
+  tab.classList.add("sa-search-profile-btn");
   tab.setAttribute("role", "tab");
   tab.setAttribute("aria-selected", false);
   tab.tabIndex = -1; // unselected tabs should only be focusable using arrow keys
   img.src = addon.self.dir + "/user.svg";
-  img.className = "tab-icon";
+  img.className = "tab-icon sa-search-profile-icon";
   span.innerText = msg("profile");
   addon.tab.displayNoneWhileDisabled(tab);
 
+  const setInvalidUsername = () => {
+    tab.disabled = true;
+    tab.title = msg("invalid-username", { username: user });
```

Comment:
The error message should now probably say that the user doesn't exist, not that the username is invalid.

Reply:
We could have 2 strings or change the current one to contemplate both cases.

`\"{username}\" is an invalid username or the account doesn't exist`

Reply:
I don't think the difference between an invalid username and a valid one that doesn't exist is very important to the user.

Reply:
Okay, I'll change the existing string